### DISCUSSION TITLE
How to use FontAwesome glyphs in NerdFonts?

### DISCUSSION THREAD

**Discussion by Mazoch** [ryanoasis/nerd-fonts#1401] [Q&A] [answered]
Hello, I have such a silly question. How to use Font Awesome glyphs after installing Nerd Fonts? They are integrated or not? From what I looked in Font Awesome cheet sheet there are very many icons missing. So I assume it's me doing something wrong.

I read the readme in github but it doesn't tell me much :/

**Comment by Finii** [comment, marked as answer]
Probably the problem is that Nerd Fonts uses Font Awesome v4.7.0 while you maybe checked their website (i.e. v6.4.2).

To see which icons Nerd Font has use _our_ cheat sheet: https://www.nerdfonts.com/cheat-sheet

See also
* #789

_(We should document the used source set versions, maybe at https://github.com/ryanoasis/nerd-fonts/wiki/Glyph-Sets-and-Code-Points, but that seems to be missing.)_

**Reply by Mazoch** [reply]
Now everything is clear, thank you for the answer :)
